code/02_create-figures.R

```diff
@@ -94,6 +94,7 @@ fig2 <-
                    fill = qom),
                colour = "white",
                size = 0.3) +
+  scale_fill_continuous(trans = "reverse") +
```

Comment:
I found a solution to make the legend appear as we want. Instead of using trans = "reverse", we can use

```
scale_fill_continuous(low = "#56B1F7", high = "#132B43")
```

In this way, the legend scale won't be reversed and the colour will show as we want (highest health board in darker blue).